src/dune

```diff
@@ -68,7 +68,7 @@
  (name main)
  (modules main mainFeature ciloptions)
  (modes native)
- (libraries goblint-cil dynlink findlib)
+ (libraries goblint-cil dynlink findlib domain_shims)
```

Comment:
The `main` executable doesn't use domain_shims directly though? I think only the goblint-cil library does, so the dependency comes transitively.